extensions/applicationinsights-clickanalytics-js/src/ClickAnalyticsPlugin.ts

```diff
@@ -99,13 +150,41 @@ export class ClickAnalyticsPlugin extends BaseTelemetryPlugin {
                     asyncCallback && asyncCallback();
                 })
             };
+
+            function _populateDefaults(config: IConfiguration) {
+                let core = _self.core;
+
+                _self._addHook(onConfigChange(config, (details) => {
+                    let config = details.cfg;
+                    let ctx = createProcessTelemetryContext(null, config, core);
+                    let _config = ctx.getExtCfg(_identifier, defaultValues);
+
+                    let logger = _self.diagLog();
+                    _contentHandler = new DomContentHandler(_config, logger);
+                    let metaTags = _contentHandler.getMetadata();
+                    _pageAction = new PageAction(this, _config, _contentHandler, _config.callback.pageActionPageTags, metaTags, logger);
+    
+                    // Default to DOM autoCapture handler
+                    _autoCaptureHandler = _autoCaptureHandler ? _autoCaptureHandler : new AutoCaptureHandler(_self, _config, _pageAction, logger);
```

Comment:
This might be problematic, as if any config changes the _pageAction will be assigned a new object, but this won't get reassigned.